src/appleseed.python/studio/__init__.py

```diff
@@ -26,4 +26,10 @@
 # THE SOFTWARE.
 #
 
+import os
+
+# Prevent Qt.py importing PySide2 and PyQt5.
+if not os.getenv("QT_PREFERRED_BINDING"):
+    os.environ["QT_PREFERRED_BINDING"] = "PyQt4"
```

Comment:
I think this is not completely correct. It ignores PySide1 which is a valid binding.

# Prevent Qt.py importing PySide2 and PyQt5.
if not os.getenv("QT_PREFERRED_BINDING"):
    os.environ["QT_PREFERRED_BINDING"] = os.pathsep.join(["PySide", "PyQt4"])